.github/workflows/default.yml

```diff
@@ -342,3 +343,47 @@ jobs:
           LINKS_MYSQL_PASSWORD: links
           LINKS_MYSQL_PORT: ${{ job.services.mysql.ports[3306] }}
         shell: bash
+
+  end-to-end:
```

Comment:
Lets name this job "webbrowser" or something long those lines (maybe client even?)

Reply:
Done